src/components/Card/BrandCard.tsx

```diff
@@ -1,16 +1,17 @@
+import { Card } from 'antd';
 import { brandData } from './constants';
 import styles from './card.module.css';
 
 export default function BrandCard() {
   return (
     <>
       {brandData.map((item) => (
-        <div
+        <Card
           className={`${styles.cardConatiner} ${styles.cardBrandConatiner} `}
```

Comment:
lets add [classnames](https://www.npmjs.com/package/classnames) to the project and use it in thoose cases